Sources/SwiftDocC/Model/Rendering/Content/RenderBlockContent.swift

```diff
@@ -76,7 +76,10 @@ public enum RenderBlockContent: Equatable {
     
     /// A video with an optional caption.
     case video(Video)
-
+    
+    /// A horizontal rule.
+    case thematicBreak
```

Comment:
I think we need a new entry in the render node spec for this (which would also bump the patch component of the version):

```json
"ThematicBreak": {
    "required": [
        "type",
    ],
    "type": "object",
    "properties": {
        "type": {
            "type": "string",
            "enum": [
                "thematicBreak"
            ]
        }
    }
},
```